From bb73fe024dc3b92f6e168284df7f9b7ef40dd880 Mon Sep 17 00:00:00 2001
From: Jappie Klooster <jappieklooster@hotmail.com>
Date: Wed, 2 Aug 2023 10:32:32 -0400
Subject: [PATCH] Set test suite as executable

---
 esqueleto-textsearch.cabal | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/esqueleto-textsearch.cabal b/esqueleto-textsearch.cabal
index a30576d..6590a48 100644
--- a/esqueleto-textsearch.cabal
+++ b/esqueleto-textsearch.cabal
@@ -32,7 +32,7 @@ library
   default-language: Haskell2010
   ghc-options:      -Wall -fwarn-incomplete-uni-patterns
 
-test-suite spec
+executable spec
   type:             exitcode-stdio-1.0
   ghc-options:      -Wall -fwarn-incomplete-uni-patterns
   hs-source-dirs:   test
-- 
2.40.1

